data/input_files/tests/thermal_conduction/static_solve.lua

```diff
@@ -43,6 +43,16 @@ thermal_conduction = {
     -- material parameters
     kappa = 0.5,
 
+    -- add a nonlinear source
+    nonlinear_reaction = {
+        reaction_function = function (temp)
```

Comment:
heh.. I guess this answers the question @joshessman-llnl  and I talked about a while ago. Whether nested functions worked (Josh was pretty sure they would) but more importantly whether people would want it and if we should advertise it.

Reply:
This isn't nested, it's two different functions. BUT NOW I WANT TO TRY IT.

Reply:
by nested i mean not at the global scope.  glad it worked!